tests/test_restore_names.py

```diff
@@ -1,3 +1,90 @@
 from app.restore_names import restore_names
 
-# write your tests here
+
+def test_name_none():
+    initial_list = [
+  {
+    "first_name": None,
+    "last_name": "Holy",
+    "full_name": "Jack Holy",
+  },
+  {
+    "first_name": "Mike",
+    "last_name": "Adams",
+    "full_name": "Mike Adams",
+  },
+]
+
+    restore_names(initial_list)
+    assert initial_list == [
+  {
+    "first_name": "Jack",
+    "last_name": "Holy",
+    "full_name": "Jack Holy",
+  },
+  {
+    "first_name": "Mike",
+    "last_name": "Adams",
+    "full_name": "Mike Adams",
+  },
+]
+
+
+def test_now_key_name():
```

Comment:
The same here.
You can use more informative name, for example:
`test_key_first_name_does_not_exist`